src/Directory.Build.props

```diff
@@ -5,6 +5,7 @@
     <Version>1.0.0</Version>
     <LangVersion>latest</LangVersion>
     <SignAssembly>true</SignAssembly>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Comment:
Wouldn't go so far; a warning is a warning.

Reply:
I can revert if you with, you just have the exact same configuration for main `AngleSharp` project:

https://github.com/AngleSharp/AngleSharp/blob/devel/src/Directory.Build.props#L8

Reply:
Yes its a good point, for consistency it makes sense. Let's keep it!